tests/units/classes/mock/streams/fs/file.php

```diff
@@ -412,7 +412,7 @@ public function testFseek()
 			->then
 				->boolean($fileObject->eof())->isTrue()
 				->boolean($fileObject->valid())->isFalse()
-				->string($fileObject->current())->isEmpty()
+				->variable($fileObject->current())->isEqualTo('')
```

Comment:
@Hywan not yet as I have to dig before opening an issue.

I have not enough clue for now : there are several weird behavior in HHVM with everything using stream wrappers and I wan't to gather as much things as I can before opening an issue and see it closed as "won't fix" ;)